### ISSUE TITLE
Not compatiable with react 18

### ISSUE THREAD

**Issue by jbakchr** [no23reason/react-geolocated#1040]
When trying to install the package in a React project using the latest version 18 I get the following error:

```
npm ERR! code ERESOLVE
npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR! 
npm ERR! While resolving: geo@0.1.0
npm ERR! Found: react@18.1.0
npm ERR! node_modules/react
npm ERR!   react@"^18.1.0" from the root project
npm ERR! 
npm ERR! Could not resolve dependency:
npm ERR! peer react@">= 0.11.2 < 18.0.0" from react-geolocated@3.2.0
npm ERR! node_modules/react-geolocated
npm ERR!   react-geolocated@"*" from the root project
npm ERR! 
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force, or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
```

Any idea how to fix this?

**Comment by no23reason** [comment]
Currently, the package was not tested with React 18, so its peerDependencies do not allow React 18 yet. It *should* work, so if you can, please use the `--legacy-peer-deps` switch for `npm install`. That will skip this check and should allow you to use this package. I will try to take a look into this to expand the peerDependencies as soon as I have the time.

**Comment by no23reason** [comment]
Fixed in #1048, you can use the version 4.0.0 without any flags in React 18.

**Comment by jbakchr** [comment]
Cool :)
